### DISCUSSION TITLE
Download artifacts from standalone fleet?

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#12104] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

29

### RAM

64

### Storage for /

1 tb

### Storage for /nsm

1 tb

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Is it possible for the agent to download endpoint artifacts  from standalone fleet node, so its no need to Connect to manager

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by defensivedepth** [comment]
There is an open issue for this: https://github.com/Security-Onion-Solutions/securityonion/issues/11346
